src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator, FigureDraw {
```

Comment:
Avoid using abstract classes to set behavior for classes. In this case, consider if `Figure` should be an abstract class or if interfaces alone would be sufficient to define the necessary behaviors.